solutions/CSES Array Description.mdx

```diff
@@ -62,5 +65,77 @@ int main() {
 		cout << ans;
 }
 ```
+</CPPSection>
+
+<JavaSection>
+
+```java
+
+import java.io.*;
+import java.util.*;
+
+public class cses1746 {
+	public static int MOD = (int) 1e9 + 7;
+	public static void main(String args[]) throws IOException {
+		BufferedReader r;
+		r = new BufferedReader(new InputStreamReader(System.in));
+		StringTokenizer st = new StringTokenizer(r.readLine());
+		int N = Integer.parseInt(st.nextToken());
+		int M = Integer.parseInt(st.nextToken());
+		int[][] dp = new int[N][M + 1];
```

Comment:
perhaps add a short explanation of what this dp array contains
like `dp[i][j] contains (insert whatever here)`